Library/Formula/git-town.rb

```diff
@@ -0,0 +1,40 @@
+class GitTown < Formula
+  desc "High-level command-line interface for Git"
+  homepage "http://www.git-town.com"
+  url "https://github.com/Originate/git-town/archive/v0.7.3.tar.gz"
+  sha256 "947fd6dac603dcb02467cc2969ef26db1ccdc72312b6b0d70fb8d11179eb09ba"
+
+  depends_on "dialog" => :recommended
+
+  def install
+    # Install the source
+    libexec.install Dir["src/*"]
+
+    # Symlink the executables
+    bin.install_symlink Dir["#{libexec}/git-*"]
+    bin.install_symlink "#{libexec}/helpers"
+    bin.install_symlink "#{libexec}/drivers"
+
+    # Install the man pages
+    man1.install Dir["man/man1/*"]
+  end
+
+  def caveats
+    <<-EOS.undent
+      To install the Fish shell autocompletions,
+      run "git town install-fish-autocompletion"
```

Comment:
We have simplified this to list only a few pointers to optional additional install scripts. Hope this addresses this issue.  :)